public/i18n/en.yaml

```diff
@@ -822,6 +825,9 @@ kyma-modules:
   change: Change
   change-release-channel: Change Release Channel
   change-release-channel-warning: The module's version in the fast channel is usually higher than the version in the regular channel. As downgrades are not supported, switching the channel back to regular will not affect the module's version <0>until the channel version is the same or higher.</0> To use the lower version, you must delete the module and re-add it from the <0>regular</0> channel.
+  messages:
```

Comment:
Shoudn't we put more in `messages` like `module-added: Module added` and others?